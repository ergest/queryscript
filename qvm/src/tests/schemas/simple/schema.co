export type User record {
    id number,
    org_id number,
    name string,
}

export type Event record {
    user_id number,
    description string,
    ts string,
}

export let users [User] = load_json('/dev/null');
export let events [Event] = load_json('/dev/null');
