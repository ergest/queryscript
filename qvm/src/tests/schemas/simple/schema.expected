{
    "decls": {
        "let events": [{user_id Float64, description Utf8, ts Utf8}],
        "let users": [{id Float64, org_id Float64, name Utf8}],
        "type Event": {user_id Float64, description Utf8, ts Utf8},
        "type User": {id Float64, org_id Float64, name Utf8},
    },
    "queries": [],
}